File: Programming Basics with Python/По-сложни проверки/journey.py

budget = float(input())
season = input()

destination = ""
vacation_place = ""
if season == "summer":
    vacation_place = "Camp"
    if budget <= 100:
        destination = "Bulgaria"
        spend = budget * 0.70
        spend_money = budget - spend
    elif budget <= 1000:
        destination = "Balkans"
        spend = budget * 0.60
        spend_money = budget - spend
    else:
        vacation_place = "Hotel"
        destination = "Europe"
        spend = budget * 0.10
        spend_money = budget - spend
elif season == "winter":
    vacation_place = "Hotel"
    if budget <= 100:
        destination = "Bulgaria"
        spend = budget * 0.30
        spend_money = budget - spend
    elif budget <= 1000:
        destination = "Balkans"
        spend = budget * 0.20
        spend_money = budget - spend
    else:
        vacation_place = "Hotel"
        destination = "Europe"
        spend = budget * 0.10
        spend_money = budget - spend

print(f"Somewhere in {destination}")
print(f"{vacation_place} - {spend_money:.2f}")

File: Programming Basics with Java/More complex checks/Journey.java
import java.util.Scanner;

public class Journey 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        double budget = Double.parseDouble(input.nextLine());
        String season = input.nextLine();

        String destination = "";
        String type_vacation = "";
        double percent = 0.0;

        if (season.equals("summer"))
        {
            if (budget <= 100)
            {
                destination = "Bulgaria";
                type_vacation = "Camp";
                percent = 0.3;
            }
            else if (budget <= 1000)
            {
                destination = "Balkans";
                type_vacation = "Camp";
                percent = 0.4;
            }
            else
            {
                destination = "Europe";
                type_vacation = "Hotel";
                percent = 0.9;
            }
        }
        else if (season.equals("winter"))
        {
            if (budget <= 100)
            {
                destination = "Bulgaria";
                type_vacation = "Hotel";
                percent = 0.7;
            }
            else if (budget <= 1000)
            {
                destination = "Balkans";
                type_vacation = "Hotel";
                percent = 0.8;
            }
            else
            {
                destination = "Europe";
                type_vacation = "Hotel";
                percent = 0.9;
            }
        }

        double spend_monney = budget * percent;
        System.out.printf("Somewhere in %s\n", destination);
        System.out.printf("%s - %.2f", type_vacation, spend_monney);
    }
}
